src/stream/src/executor/sink.rs

```diff
@@ -200,6 +200,7 @@ impl<F: LogStoreFactory> SinkExecutor<F> {
         let input = input.inspect_ok(move |msg| {
             if let Message::Chunk(c) = msg {
                 metrics.sink_input_row_count.inc_by(c.capacity() as u64);
+                metrics.sink_input_size.inc_by(c.estimated_size() as u64);
```

Comment:
I think the metrics here will be inaccurate when sink decoupling is on because the metrics here is measuring the throughput of the input channel. When sink decoupling is on, the input will be polled and written to log store regardless of the external sink.

We need to add a metric for log store reader throughput: https://github.com/risingwavelabs/risingwave/blob/9defad37fb0ac2dba9ac3c34b6d9bbfbc969c75e/src/connector/src/sink/log_store.rs#L306

Reply:
It looks like the metric here is log_store_read_rows, which doesn't match the sink_input_row_count we defined earlier. Should we merge the two or add another log_store_read_rows（mb/s）